src/pages/application.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import type { NextPage } from 'next';
+import ApplicationPage from 'components/ApplicationPage';
+
+const Application: NextPage = () => {
+  return (
+    <>
+      <ApplicationPage />
+    </>
+  );
+};
```

Comment:
```suggestion
const Application: NextPage = () => <ApplicationPage />;
```

더 이상 들어갈 내용이 없다면 이렇게 줄이는 것도 좋을 것 같아요 👍